scenarioo-server/src/main/java/org/scenarioo/rest/application/ScenariooWebApplication.java

```diff
@@ -35,41 +35,42 @@
  * Scenarioo REST Services Web Application context that initializes data on startup of server.
  */
 public class ScenariooWebApplication implements ServletContextListener {
-	
+
 	private static final Logger LOGGER = Logger.getLogger(ScenariooWebApplication.class);
-	
+
 	@Override
 	public void contextInitialized(final ServletContextEvent servletContextEvent) {
 		LOGGER.info("====================================================");
 		LOGGER.info("Scenarioo Viewer is starting up ...");
 		LOGGER.info("====================================================");
-		
+
 		initializeApplicationVersion(servletContextEvent.getServletContext());
 		loadConfiguration(servletContextEvent);
-		
+		initializeApplicationContext(servletContextEvent.getServletContext());
```

Comment:
I would either inline this method, or give it a more precise name, like initializeContextPath